tests/functional/spec/interviewer_note.spec.js

```diff
@@ -0,0 +1,32 @@
+import InitialInterstitialPage from "../generated_pages/interviewer_note/initial-interstitial-block.page.js";
+import FavouriteTeamPage from "../generated_pages/interviewer_note/favourite-team-block.page.js";
+import ConfirmPage from "../generated_pages/interviewer_note/confirm-block.page.js";
+import FinalInterstitialPage from "../generated_pages/interviewer_note/final-interstitial-block.page.js";
+
+describe("Interviewer Note", () => {
+  beforeEach(() => {
```

Comment:
So I wouldn't use beforeEach, before would be better. This would mean you don't repeat each step, the next 'it' will just follow on where you  left off form the last 'it' You naturally would have to move the Given I started out of each test, but it will make this test a little shorter and quicker.

Reply:
Changed, thanks!